resources/views/mail/paymentreceipt.blade.php

```diff
@@ -0,0 +1,11 @@
+Hi {{ $payment->payable->user->preferred_first_name }},
+
+@if($payment->method === "square")
+We received your online payment of ${{ number_format($payment->amount, 2) }} for {{ $payable_name }}.
+
+You can view more details about your payment at {{ $payment->receipt_url }}
```

Comment:
```suggestion
You can view more details about your payment at {{ $payment->receipt_url }}.
```

Reply:
Will need to test how that renders in real clients, but ok